semih2.js

```diff
@@ -0,0 +1,190 @@
+// #1
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length);
+
+const newString = myString.split(",").join(" ");
+console.log(newString);
+
+
+// #2
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+favoriteAnimals.push("turtle");
+console.log(favoriteAnimals);
+
+favoriteAnimals.unshift("meerkat");
+console.log(favoriteAnimals);
+console.log("The array has a length of: ", favoriteAnimals.length);
+
+favoriteAnimals.splice(3,1);
+console.log(favoriteAnimals);
+console.log("The item you are looking for is at index: ", favoriteAnimals.indexOf("meerkat"));
+
+favoriteAnimals.splice(favoriteAnimals.indexOf("meerkat"),1);
+console.log(favoriteAnimals);
+
+
+// #3
+//////////////1//////////////////////
+function addUp (x,y,z) {
+    const totalSum = (x+y+z);
+    console.log(totalSum);
+}
+addUp(1,10,100);
+/////////////////////////////////// 
+
+
+///////////////2///////////////////
+function colorCar(color) {
+    console.log("A", color, "car.");
```

Comment:
This works but can be done in some other ways, `'A ' + color + ' car.'` or using backticks `a ${color} car`

Reply:
Thanks for the alternatives! 👍  I guess I should get used to using `$` sign more often :)